static/app/views/dashboards/widgets/timeSeriesWidget/timeSeriesWidgetVisualization.tsx

```diff
@@ -264,12 +295,30 @@ export function TimeSeriesWidgetVisualization(props: TimeSeriesWidgetVisualizati
     return getFormatter({
       isGroupedByDate: true,
       showTimeInTooltip: true,
-      valueFormatter: (value, field) => {
-        if (!field) {
-          return formatTooltipValue(value, FALLBACK_TYPE);
+      valueFormatter: function (value, _field, valueFormatterParams) {
+        // Use the series to figure out the corresponding `Plottable`, and get the field type. From that, use whichever unit we chose for that field type.
+
+        if (!valueFormatterParams || !defined(valueFormatterParams?.seriesIndex)) {
+          // The series might be missing is this is a formatter for a mark line.
```

Comment:
```suggestion
          // The series might be missing as this is a formatter for a mark line.
```

I think this is what you meant?

Reply:
Oh, I meant "if" here, fixed!